### ISSUE TITLE
如何将两个Dataset同时用在一个训练上

### ISSUE THREAD

**Issue by Tuziking** [PaddlePaddle/PaddleOCR#10084]
在AIstudio上运行
将MNIST反色后和原来的数据集合并用在fit中报错
import paddle
from paddle.io import Dataset

class combine_DataSet(Dataset):
    def __init__(self, dataset1, dataset2):
        print("haha")
        self.dataset1 = dataset1
        self.dataset2 = dataset2

    def __len__(self):
        return len(self.dataset1) + len(self.dataset2)

    def __getitem__(self, idx):
        if idx < len(self.dataset1):
            return self.dataset1[idx]
        else:
            return self.dataset2[idx - len(self.dataset1)]

class Invert(object):
    def __init__(self):
        pass

    def __call__(self, img):
        return 255 - img

invert_transform = T.Compose([T.Resize((28, 28)), T.ToTensor(), Invert(), T.Normalize(mean=[127.5], std=[127.5])])
transform = T.Normalize(mean=[127.5], std=[127.5])

# 训练数据集
train_invert_dataset = MNIST(mode='train', transform=invert_transform)
train_ori_dataset = MNIST(mode='train', transform=transform)

# 验证数据集
test_ori_dataset = MNIST(mode='test', transform=invert_transform)
test_invert_dataset = MNIST(mode='test', transform=transform)

# 合并白底黑字数据集与黑底白字数据集
train_dataset = combine_DataSet(train_invert_dataset, train_ori_dataset)
test_dataset = combine_DataSet(test_invert_dataset, test_ori_dataset)

**Comment by Gmgge** [comment]
可以在配置文件中设置，例如训练检测模型时，使用[configs/det/det_mv3_db.yml](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/configs/det/det_mv3_db.yml#L70C5-L70C15)配置文件（也可以在选择其他配置文件）进行训练，示例操作如下：
1.修改其69行label_file_list的值为列表形式，[data1_label.txt, data2_label.txt];
2.修改70行，数据集采样比例，可以设置为[0.5, 0.5]。

更多的参数说明可以参考该[配置](https://github.com/PaddlePaddle/PaddleOCR/blob/df064f45386340ad648bafd9cfe87f8d89c4ddff/doc/doc_ch/config.md)文件
